monitoring_service.rst

```diff
@@ -2,145 +2,230 @@ Raiden Network Monitoring Service
-* If MS registry is used, a deposit will be required for registration
+Summary
+=======
+*   Monitoring Service (MS) listens to blinded Balance Proofs (BP) and proposed service fees in a public Matrix Room
+*   Any pre-registered MS can decide to monitor a channel and store corresponding BPs
+*   Then, whenever a channel is closed by calling `closeChannel` the MS will call `updateNonClosingBalanceProof` with
+    the submitted BP by its client
+*   Service fees are paid in RDN, there is no free tier
+*   In the short term we go for a simple design which will allow us to reach the Ithaca milestone earlier
+*   In the long-term we see the PISA approach as more economically viable and user friendly, but this design requires
+    additional features and can be developed independently after Ithaca
 
 Usual scenario
 ==============
 
-The Raiden node that belongs to Alice is going offline and Alice wants to be protected against having her channels closed by Bob with an incorrect :term:`balance proof`.
+The Raiden node that belongs to Alice is going offline and Alice wants to be protected against having her channels
+closed by Bob with an incorrect `balance proof`.
 
-1) Alice broadcasts the balance proof by sending a message to a public chat room.
-2) Monitoring services decide if the fee is worth it and picks the balance proof up.
-3) Alice now goes offline.
-4) Bob sends an on-chain transaction in attempt to use an earlier balance proof that is in his favor.
-5) Some of the monitoring servers detect that an incorrect :term:`BP` is being used. They can update the channel closing state as long as the :term:`Challenge Period` is not over.
-6) After the Challenge period expires, the channel can be settled with a balance that Alice expects to be correct.
+1)  Whenever Alice wants to get her channel monitored, Alice publishes her Monitor Request - blinded balance proofs -
+    into a public chat room
+2)  Updates can be sent whenever the state or proposed fee changes
+3)  MSs can pick up these messages and then listen to closeChannel events regarding this particular channel
+4)  If so, whenever the channel is closed and before the settlement period has ended, the MS will call the
+    `updateNonClosingBalanceProof` function with the provided information. This way the settleChannel function that
+    calculates the token distribution can only be executed submitting the corresponding balance values of the hashed
+    balances provided before
+5)  Either Alice or Bob (or anyone else) can now anytime call settleChannel and initiate the token distribution
+6)  This will result in a correct distribution of tokens, in accordance with the off-chain transfers
+7)  After a successful submission of the hashed balance by the MS (might be a defeat of an attack), the monitoring
+    service gets an on-chain payment from a smart contract where Alice deposited some tokens beforehand
```

Comment:
I'd also mention which function in the SC is used for this.

Reply:
done